doc/source/user/quickstart.rst

```diff
@@ -1441,7 +1437,7 @@ To change the dimensions of an array, you can omit one of the sizes
 which will then be deduced automatically::
 
     >>> a = np.arange(30)
-    >>> a.shape = 2,-1,3  # -1 means "whatever is needed"
+    >>> a = a.reshape((2, -1, 3))  # -1 means "whatever is needed"
```

Comment:
maybe change the returned value's name to make it clear this is a new object: `b = a.reshape(...`